Sources/Utils/Rng/MobileCoinSeedableRng.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+// swiftlint:disable unused_setter_value
+import Foundation
+
+public class MobileCoinSeedableRng: MobileCoinRng {
+    let _seed: Data32
+
+    var seed: Data32 {
+        _seed
+    }
+
+    init(seed: Data32) {
+        self._seed = seed
+    }
+
+    override convenience init() {
+        let bytesCount = 32
```

Comment:
first part of this code looks the same as `func securityRNG(...)` maybe just re-use that instead to avoid duplication ?

Reply:
The `func securityRNG(...)` func only returns a UInt64 / 8 bytes, and has a `context` param that's really only relevant for it's use as a callback...   But maybe we can refactor the implementation of securityRNG AND this code to both use some convenience function(s) to generate data of a specified size and have them both call that function... that way the RNG generation is common between both.  Will give it a shot.

Reply:
DRY'ed this out via an extension on Data to generate a specified number of random bytes of data, and plugged that in for both the default func for the UInt64 and for the generation of the 32 byte seed for ChaCha20